ros_controls.humble.repos

```diff
@@ -1,8 +1,4 @@
 repositories:
-  ros-controls/gazebo_ros2_control:
-    type: git
-    url: https://github.com/ros-controls/gazebo_ros2_control.git
-    version: humble
```

Comment:
Why did we remove this now?

Reply:
I just updated the description. I was too lazy to open a separate PR. should I, or change the title here?

Reply:
Got it